queries/ecma/injections.scm

```diff
@@ -57,6 +57,16 @@
 ((glimmer_template) @injection.content
   (#set! injection.language "glimmer"))
 
+; css`<css>`, keyframes`<css>`
+(call_expression
+  function: (identifier) @_name
+  (#match? @_name "^(css|keyframes)")
```

Comment:
```suggestion
  (#any-of? @_name "css" "keyframes")
```

Reply:
Yes and no, it is injected as part of the more generalized injection for html, gql, etc. However the purpose of using `css` here is to inject with the `styled` grammar which supports JS interpolation which is very common in CSS-in-JS frameworks:

```js
const style = css`
  padding: 32px;
  border: 1px solid ${color};
`
```